Anlab.Mvc/Client/css/components.scss

```diff
@@ -1,4 +1,5 @@
 //images
+@import "~react-datepicker/dist/react-datepicker.css";
```

Comment:
@laholstege just to see, try this without the ~ and instead use ./

want to see if that works

Reply:
@srkirkland doesn't work without the ~